_static/css/aiosmtpd.css

```diff
@@ -0,0 +1,4 @@
+.boldital {
+    font-weight: bold;
+    font-style: italic;
+}
```

Comment:
I still find it incredulous that reStructuredText does not support **_bolded italics_** out-of-the-box and had to resort to CSS trickeries...